backend/src/main/java/com/example/backend/Controllers/NotificationController.java

```diff
@@ -0,0 +1,29 @@
+package com.example.backend.Controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import com.example.backend.DTOs.NotificationDTO;
+import com.example.backend.DTOs.PageResponse;
+import com.example.backend.Services.NotificationService;
+
+import lombok.RequiredArgsConstructor;
+
+@Controller
+@RequestMapping("/notifications")
+@RequiredArgsConstructor
+@CrossOrigin(origins = "http://localhost:3000")
```

Comment:
no need to define the origin here as this url may be changed just use 
@CrossOrigin(origins = "*")
as usual.